sycl/include/CL/sycl/group.hpp

```diff
@@ -264,21 +265,27 @@ template <int dimensions = 1> class group {
   device_event async_work_group_copy(local_ptr<dataT> dest,
                                      global_ptr<dataT> src,
                                      size_t numElements) const {
-    __ocl_event_t e =
-        OpGroupAsyncCopyGlobalToLocal<dataT>(
-            __spv::Scope::Workgroup,
-            dest.get(), src.get(), numElements, 1, 0);
+    using T = detail::ConvertToOpenCLType_t<dataT>;
+    using Local = detail::ConvertToOpenCLType_t<local_ptr<dataT>>;
```

Comment:
```suggestion
    using Local = detail::ConvertToOpenCLType_t<decltype(dest)>;
```
Not sure whether it's better or not.